src/components/PostList/PostList.tsx

```diff
@@ -1,5 +1,17 @@
 import React from 'react';
+import { PostWithAllDate } from '../../types/postWithAllDate';
+import { PostInfo } from '../PostInfo';
 
-export const PostList: React.FC = () => (
-  <>Put the list here</>
+type Props = {
+  posts: PostWithAllDate[]
+};
+
+export const PostList: React.FC<Props> = ({ posts }) => (
+  <ul className="PostList">
+    {
+      posts && posts.map((object: PostWithAllDate) => (
+        <PostInfo post={object} key={object.id} />
+      ))
+    }
```

Comment:
in your Props you always will have posts
but, if you want to check that array is not empty - use 
post.length > 0 or !!post.length

```suggestion
    {posts.map((object: PostWithAllDate) => (
        <PostInfo post={object} key={object.id} />
      ))}
```